pyproject.toml

```diff
@@ -55,6 +55,7 @@ pygments-github-lexers = "^0.0.5"
 black = { version = "^18.3-alpha.0", python = "^3.6" }
 pre-commit = "^1.10"
 tox = "^3.0"
+mock = "^2.0"
```

Comment:
This is not necessary since we already have `pytest-mock` which provides a more convenient way to mock objects and functions.